luigi_pipeline/seqr_loading_optimized.py

```diff
@@ -24,6 +24,8 @@ class SeqrVCFToVariantMTTask(SeqrVCFToMTTask):
 
     def run(self):
         # We only want to use the Variant Schema.
+        if self.remap_path: check_if_path_exists(self.remap_path, "remap_path")
```

Comment:
Yeah, we could, that seems cleaner -- I'm not sure what the impact on runtime will be but seems like it should be quick. I think for the WGS remap issue we keep hitting though, I updated the airflow code in a way that always expects a remap path because I didnt expect DSP to go revert to their old pipeline https://github.com/broadinstitute/seqr-pipeline-airflow/blob/1e5419b1ca2e7755e99c1c04f4b85666213161d5/airflow/dags/dependencies/luigi_task_args.py#L67. But ideally, I should've made that optional like it is at the project level: https://github.com/broadinstitute/seqr-pipeline-airflow/blob/1e5419b1ca2e7755e99c1c04f4b85666213161d5/airflow/dags/seqr-vcf-to-es.py#L451